qa/mc_sc_connected_nodes.py

```diff
@@ -24,19 +44,28 @@ def setup_network(self, split = False):
         self.sync_all()
 
     def setup_nodes(self):
-        return start_nodes(self.number_of_mc_nodes, self.options.tmpdir)
+        return start_nodes(self.number_of_mc_nodes, self.options.tmpdir, extra_args=[["-websocket"],["-websocket"],["-websocket"]])
 
     def sc_setup_chain(self):
-        network = {
-            self.nodes[0]:{
-                0: [1, 1000, create_websocket_configuration("ws://localhost:{0}".format(self.ws_port_mc_0))],
-                1: [2, 1000, create_websocket_configuration("ws://localhost:{0}".format(self.ws_port_mc_0))],
-            },
-            self.nodes[1]:{
-                0: [3, 1000, create_websocket_configuration("ws://localhost:{0}".format(self.ws_port_mc_1))]
-            }
-        }
-        self.bootstrap_sidechain(network)
+        mc_node_1 = self.nodes[0]
+        mc_node_2 = self.nodes[1]
+        sc_node_1_configuration = SCNodeConfiguration(
+            mc_node_1,
+            SCCreationInfo("1".zfill(64), 100, 1000),
```

Comment:
`SCCreationInfo("1".zfill(64), 100, 1000)` and `SCCreationInfo("2".zfill(64), 250, 1000)` how they can be different?
Does it mean, that multiple sidechains were declared in a mc network, and each sc node follow it's own sidechain?

Reply:
Changed. Now is not different